authentication/jsonld-contexts/authentication.jsonld

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
`authentication.jsonld` is not the more appropriate name for this context (actually, it deals with authorization and not with authentication).

so ... `authorization.jsonld` or a more extensible `security.jsonld`?

Reply:
authorization sounds nice to me